lita.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency "bundler", "~> 2.0"
-  spec.add_runtime_dependency "faraday", "~> 0.15.0"
+  spec.add_runtime_dependency "faraday", "~> 1.0"
```

Comment:
Addressed in https://github.com/litaio/lita/commit/9bb5951dd9a71f81b4f2ce4f194869d13765ccc3#diff-ee0dfbe0509abc47c944702671133788R24